src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -288,11 +336,27 @@ static object ParseValueObject(IInputObjectGraphType graphType, string variableN
                 // must be thrown. In either case, the input object literal or unordered
                 // map must not contain any entries with names not defined by a field
                 // of this input object type, ***otherwise an error must be thrown.***
+                //List<string> unknownFields = null;
+                //foreach (var key in dic.Keys)
+                //{
+                //    bool match = false;
+
+                //    foreach (var key2 in graphType.Fields.List)
+                //    {
+                //        if (key == key2.Name)
+                //        {
+                //            match = true;
+                //            break;
+                //        }
+                //    }
+
+                //    if (!match) (unknownFields ??= new List<string>(1)).Add(key);
```

Comment:
Nice implementation.

Reply:
Oh - lol - I didn't finish it up obviously.  I was having trouble with tests, and then I guess I never went back to it 🤦‍♂️

Reply:
I think it's all done; I'll remove the LINQ as long as it's working properly.

Reply:
Done.